tests/xrt/56_xclbin/kernel.cl

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2016-2017 Xilinx, Inc
```

Comment:
Do these years make sense?

Reply:
Fixed. This was copy and paste issue. Looks like all XRT tests have this same problem:
https://github.com/Xilinx/XRT/blob/master/tests/xrt/00_hello/kernel.cl